### ISSUE TITLE
Update to v3 endpoints

### ISSUE THREAD

**Issue by jaggedsoft** [jaggedsoft/node-binance-api#333]
See changelog: https://github.com/binance-exchange/binance-official-api-docs/blob/master/CHANGELOG.md

**Comment by gunar** [comment]
I don't think we use any of these endpoints but we should start a task-force to update all v1 endpoints in use into using v3 endpoints. @jaggedsoft are you keeping track of that in an issue somewhere?

**Comment by andreiashu** [comment]
just an fyi there's still 2 results upon a `api/v1` [search](https://github.com/jaggedsoft/node-binance-api/search?q=api%2Fv1&unscoped_q=api%2Fv1) on this repo. There's still a call to api/v1/time endpoint.

**Comment by bmino** [comment]
This can be broken down into two chunks of work:
1. Endpoints that will not be migrated to v3 and instead require new endpoints
2. Endpoints that are simply being migrated from v1 to v3

Thankfully we have already updated the two endpoints that require new endpoints (1), all good. However, we are currently using ALL of the soon to be removed v1 endpoints (2) and this needs to be fixed asap!

**By end of Q1 2020, the following endpoints will be removed from the API.**
```
* GET api/v1/depth
* GET api/v1/historicalTrades
* GET api/v1/aggTrades
* GET api/v1/klines
* GET api/v1/ticker/24hr
* GET api/v1/ticker/price
* GET api/v1/exchangeInfo
* POST api/v1/userDataStream
* PUT api/v1/userDataStream
* GET api/v1/ping
* GET api/v1/time
* GET api/v1/ticker/bookTicker
```

**Comment by bmino** [comment]
This was fixed in #429
